### PR TITLE
Treelist footer row styling

### PR DESCRIPTION
**Pull request by inikolova**
Closes https://github.com/telerik/kendo-themes/issues/1118

### CODE REVIEW

File: packages/material/scss/treelist/_layout.scss [hunk 1]
```diff
@@ -1 +1,17 @@
 @import "~@progress/kendo-theme-default/scss/treelist/_layout.scss";
+
+@include exports("treelist/layout/material") {
+    .k-treelist {
+        tr.k-footer-template td {
+            border-width: 1px 0;
```
Comment (PreslavKozovski) [inline packages/material/scss/treelist/_layout.scss:6]: This style is causing a double border - 

image

File: packages/material/scss/treelist/_layout.scss [hunk 2]
```diff
@@ -1 +1,17 @@
...
+        }
+
+        tr.k-footer td {
```
Comment (PreslavKozovski) [inline packages/material/scss/treelist/_layout.scss:9]: I cannot see a case where we use this selector. Could you please show me an example in the TreeList? 

If we are using it, as the CSS is the same, we can add the selector to the next selector.

File: packages/material/scss/treelist/_layout.scss [hunk 3]
```diff
@@ -1 +1,17 @@
...
+            border-width: 0 0 1px 0;
+        }
+
+        tr.k-footer-template + tr.k-footer-template td {
+            border-width: 0 0 1px 0;
```
Comment (PreslavKozovski) [inline packages/material/scss/treelist/_layout.scss:14]: This could be a variable

File: packages/default/scss/treelist/_layout.scss [hunk 4]
```diff
@@ -35,6 +35,17 @@
...
         }
+
+        tr.k-footer td,
+        tr.k-footer-template td {
+            border-width: 1px 0;
+            border-style: solid;
+            font-weight: bold;
+        }
+
+        tr.k-footer + tr.k-footer td {
```
Comment (PreslavKozovski) [inline packages/default/scss/treelist/_layout.scss:46]: ```tr.k-footer-template + tr.k-footer-template td ```

Should resolve the double border problem - 

image

File: packages/material/scss/treelist/_layout.scss (hunk 2 above)
Comment (inikolova) [reply packages/material/scss/treelist/_layout.scss:9]: This selector is used in the Kendo UI TreeList for Angular, however I will add it to the next one :)

File: packages/default/scss/treelist/_layout.scss (hunk 4 above)
Comment (inikolova) [reply packages/default/scss/treelist/_layout.scss:46]: Adding this rule will remove the top border from every footer row in jQuery TreeList:

Screenshot 2020-06-30 at 13 03 23

In jQuery TreeList all aggregates are initially rendered, however their visibility is toggled via **display: none;**

File: packages/material/scss/treelist/_layout.scss (hunk 1 above)
Comment (inikolova) [reply packages/material/scss/treelist/_layout.scss:6]: According to the PSD file, there are borders: 
https://github.com/telerik/kendo-ux/blob/master/themes/default/PSD/treelist.psd

**Comment by silviyaboteva** [comment]
Only in the React tree list test, please add k-group-footer class to the inner group footer rows as like this: 
Screenshot 2020-07-22 at 11 41 20

**Pull request merged**